CMakeLists.txt

```diff
@@ -2,13 +2,9 @@
 #   intro: https://codingnest.com/basic-cmake/
 #   best practices (3.0+): https://gist.github.com/mbinna/c61dbb39bca0e4fb7d1f73b0d66a4fd1
 
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
not sure if it matters, but this also shows up in:

```
third-party/CMakeLists.txt|2| cmake_minimum_required (VERSION 2.8.7)
third-party/cmake/GettextCMakeLists.txt|1| cmake_minimum_required(VERSION 2.8.7)
third-party/cmake/GperfCMakeLists.txt|1| cmake_minimum_required(VERSION 2.8.7)
third-party/cmake/LibiconvCMakeLists.txt|1| cmake_minimum_required(VERSION 2.8.7)
third-party/cmake/LibuvCMakeLists.txt|1| cmake_minimum_required(VERSION 2.8.7)
third-party/cmake/Libvterm-tbl2inc_c.cmake|1| cmake_minimum_required(VERSION 2.8.7)
third-party/cmake/LibvtermCMakeLists.txt|1| cmake_minimum_required(VERSION 2.8.11)
third-party/cmake/UnibiliumCMakeLists.txt|1| cmake_minimum_required(VERSION 2.8.7)
third-party/cmake/libtermkeyCMakeLists.txt|1| cmake_minimum_required(VERSION 2.8.7)
```

Reply:
Ah, right.  I missed third-party somehow. :)